resources/assets/js/components/JobPreview.tsx

```diff
@@ -68,67 +150,183 @@ const JobPreview: React.FunctionComponent<JobPreviewProps> = ({
         </div>
+        <div data-c-grid-item="base(1of1)">
+          <p data-c-colour="c1" data-c-margin="bottom(quarter)">
+            <FormattedMessage
+              id="jobPreview.education"
+              defaultMessage="Education"
+              description="Datum Label"
+            />
+          </p>
+          <p>{education}</p>
+        </div>
+      </div>
+      <h4
+        data-c-border="bottom(thin, solid, black)"
+        data-c-font-size="h4"
+        data-c-font-weight="600"
+        data-c-margin="top(double) bottom(normal)"
+        data-c-padding="bottom(normal)"
+      >
+        <FormattedMessage
+          id="jobPreview.workStyles"
+          defaultMessage="Work Styles"
+          description="Section Header"
+        />
+      </h4>
+      <div data-c-grid="gutter">
         <div data-c-grid-item="tp(1of2)">
           <p data-c-colour="c1" data-c-margin="bottom(quarter)">
-            Government Classification
+            <FormattedMessage
+              id="jobPreview.remoteWork"
+              defaultMessage="Remote Work"
+              description="Datum Label"
+            />
           </p>
           <p>
-            {classification}-{level}
+            {remoteWork
+              ? intl.formatMessage(messages.allowed)
+              : intl.formatMessage(messages.notAllowed)}
+            {/* TODO: Add long form text explanation of Remote Work Status */}
+            {/* eg. Yes, I’m willing to supervise employees in any province or territory
+            in Canada. */}
+          </p>
+        </div>
+        <div data-c-grid-item="tp(1of2)">
+          <p data-c-colour="c1" data-c-margin="bottom(quarter)">
+            <FormattedMessage
+              id="jobPreview.telework"
+              defaultMessage="Telework"
+              description="Datum Label"
+            />
+          </p>
+          <p>{intl.formatMessage(messages[telework])}</p>
+        </div>
+        <div data-c-grid-item="tp(1of2)">
+          <p data-c-colour="c1" data-c-margin="bottom(quarter)">
+            <FormattedMessage
+              id="jobPreview.flexibleHours"
+              defaultMessage="Flexible Hours"
+              description="Datum Label"
+            />
+          </p>
+          <p>{intl.formatMessage(messages[flexHours])}</p>
+        </div>
+        <div data-c-grid-item="tp(1of2)">
+          <p data-c-colour="c1" data-c-margin="bottom(quarter)">
+            <FormattedMessage
+              id="jobPreview.travel"
+              defaultMessage="Travel"
+              description="Datum Label"
+            />
+          </p>
+          <p>Not yet implemented.</p>
+        </div>
+        <div data-c-grid-item="tp(1of2)">
+          <p data-c-colour="c1" data-c-margin="bottom(quarter)">
+            <FormattedMessage
+              id="jobPreview.overtime"
+              defaultMessage="Overtime"
+              description="Datum Label"
+            />
           </p>
+          <p>Not yet implemented.</p>
```

Comment:
This should be localized and/or conditionally shown based on the prop.

Reply:
Agreed, conditionally shown on the prop. 👍